docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
+**Q: I want to see the car entry boxes and car sell orders related to me (both created by me and proposed to me).**
 
-*A:* Implement new storage that will be managed by the application state to store this information. Implement a new HTTP API, that contains a new method to get this information.
+*A:* Implement new storage that will be managed by the application state (NOTE: item 3 has this as "ApplicationState"; please make consistent) to store this information. Implement a new HTTP API, that contains a new method to get this information. (Should the immediately preceding sentence instead be "Implement a new HTTP API call that contains a method to get this information."?)
 
-So, the starting point of the development process is the data representation. A car is an example of a non-coin box because it represents some entity, but not money. Another example of a non-coin box is a car that is selling. We need another box for a selling car because a standard car box does not have additional data like sale price, seller proposition address, etc. For the money representation, standard Regular Box is used (Regular box is coin box), SDK provides that box. Besides new entities CarBox and CarSellOrder, we also need to define a way to create/destroy those new entities. For that purpose, new transactions shall be defined: transaction for creating a new car, a transaction that moves CarBox to CarSellOrder, transaction which declares car selling, i.e., moving CarSellOrder to the new CarBox. All created transactions are not automatically put into the memory pool, so a raw transaction in hex representation shall be put by /transaction/sendTransaction API request. In summary, we will add the next car boxes and transactions:
+So, the starting point of the development process is the data representation. A car is an example of a non-coin box because it represents some entity, but not money. Another example of a non-coin box is a car that is selling. We need another box for a selling car because a standard car box does not have additional data like sale price, seller proposition address, etc. For the money representation, a standard Regular Box is used (a Regular box is a coin box), which the SDK provides. Besides new entities CarBox and CarSellOrder, we also need to define a way to create/destroy those new entities. For that purpose, new transactions shall be defined: transaction for creating a new car, a transaction that moves CarBox to CarSellOrder, transaction which declares car selling, i.e., moving CarSellOrder to the new CarBox. All created transactions are not automatically put into the memory pool, so a raw transaction in hex representation shall be put by /transaction/sendTransaction API request. In summary, we will add the next car boxes and transactions: (Consistency needed regarding "Car Box" vs "CarBox" and "Car Sell Order" vs "CarSellOrder" if they refer to the same thing. I imagine the API's syntax should take precedence.)
```

Comment:
@zennermac 

The starting point of the development process is the data representation. A car is an example of a Non-CoinBox. It represents an item, but not money. Another example of a Non-CoinBox is a car that is for sale. We need another box for a car for sale because a standard CarBox does not have additional data like sale price, seller proposition address, etc. For the money representation, a standard RegularBox is used (a RegularBox is a CoinBox), which the SDK provides. Besides new entities CarBox and CarSellOrder, we also need to define a way to create/destroy those new entities. For that purpose, these new transactions are defined: a transaction for creating a new car, a transaction that moves a CarBox to a CarSellOrder, a transaction that declares a car was purchased i.e., moving CarSellOrder to the new CarBox. All created transactions are not automatically put into the memory pool, so a raw transaction in hex representation is created with the /transaction/sendTransaction API request. In summary, we will add the next car boxes and transactions: